Change styling for all pages

Myshro opened the pull request.

Myshro committed:
change styling

Myshro committed:
change styling for all pages

smayya337 commented:
<img width="1512" alt="Screenshot 2024-05-11 at 10 50 40 AM" src="https://github.com/CNS-UVA/uvacns-website/assets/20958914/60fb7014-8264-4ce3-8c8c-7698429e53f9">
very nice

smayya337 commented:
More seriously, this looks pretty good! A couple quick changes:

* I'm not huge on the orange "Computer and Network Security Club @ UVA" in the header -- I think white will be fine with orange on hover
* Maybe just make an SVG of the icon instead of having to upscale a PNG? I think this might be an old Font Awesome icon (from FA4?) so there's likely an existing SVG out there somewhere

Myshro committed:
add svg for shield, change color of title, make site usable on mobile

So I ran into an issue for the SVG where I only wanted to use just one old FA4 icon (the shield), but to do so I had to install this entire directory. This is probably a bad way to do things but idk, lmk if it's an issue (it probably is)

Also, the site should be useable on smaller screens now.

Myshro commented:
Got you, read my thing about SVG in the PR, I'm sure there's a better way to do the SVG than the way I did it.
